conda/environments/cudf_dev_cuda10.0.yml

```diff
@@ -48,6 +48,7 @@ dependencies:
   - hypothesis
   - pip:
       - sphinx-markdown-tables
+      - sphinx-copybutton
```

Comment:
These are both now in conda, can we move them up from pip? 😄

Reply:
Done 👍